analyses/fusion-frequencies/README.md

```diff
@@ -16,26 +16,35 @@ Each gene invloved in the fusion is annotated by a Gene_Position, for example:
 - `Frequency_in_overall_dataset`:
-  - For each unique variant, count the number of patients (identified by `Kids_First_Participant_ID`) that have the variant, and call this number `Total_alterations`.
+  - For each unique fusion, count the number of patients (identified by `Kids_First_Participant_ID`) that have the fusion, and call this number `Total_alterations`.
   - Count the total number of patients in the `cancer_group_cohort`, and call this number `Patients_in_dataset`.
   - `Frequency_in_overall_dataset = Total_alterations / Patients_in_dataset`.
 
 - `Frequency_in_primary_tumors`:
-  - For each unique variant, count the number of samples (identified by `Kids_First_Biospecimen_ID`) that are in the `../independent-samples/results/independent-specimens.rnaseq.primary.eachcohort.tsv`, and call this number `Total_primary_tumors_alterated`.
+  - For each unique fusion, count the number of samples (identified by `Kids_First_Biospecimen_ID`) that are in the `../independent-samples/results/independent-specimens.rnaseq.primary.eachcohort.tsv`, and call this number `Total_primary_tumors_alterated`.
   - Count the total number of samples in the `cancer_group_cohort` that are also in the `../independent-samples/results/independent-specimens.rnaseq.primary.eachcohort.tsv`, and call this number `Primary_tumors_in_dataset`.
   - `Frequency_in_primary_tumors = Total_primary_tumors_alterated / Primary_tumors_in_dataset`.
 
 - `Frequency_in_relapse_tumors`:
-  - For each unique variant, count the number of samples (identified by `Kids_First_Biospecimen_ID`) that are in the `../independent-samples/results/independent-specimens.rnaseq.relapase.eachcohort.tsv`, and call this number `Total_relapse_tumors_alterated`.
+  - For each unique fusion, count the number of samples (identified by `Kids_First_Biospecimen_ID`) that are in the `../independent-samples/results/independent-specimens.rnaseq.relapase.eachcohort.tsv`, and call this number `Total_relapse_tumors_alterated`.
   - Count the total number of samples in the `cancer_group_cohort` that are also in the `../independent-samples/results/independent-specimens.rnaseq.relapase.eachcohort.tsv`, and call this number `Relapse_tumors_in_dataset`.
   - `Frequency_in_relapse_tumors = Total_relapse_tumors_alterated / Relapse_tumors_in_dataset`.
 
 Format the SNV mutation frequency table according to the latest spreadsheet that is attached in <https://github.com/PediatricOpenTargets/ticket-tracker/issues/64>.
 
 Merge the fusion frequency tables of all `cancer_group_cohort`s.
+The combined table was then written out as `putative-oncogene-fusion-freq.jsonl.gz`.
+
+Table two - `putative-oncogene-fused-gene-freq.jsonl.gz`
+The overall calculation logic is identical to table one. 
+The difference is that: instead of calculating the frequency for each unique fusion, we are calculating the frequency for each unique gene that are being fused (as either 1A, 1B, 2A or 2B in a particular fusion) in different tumor types.
```

Comment:
```suggestion
The combined table was is written out as `putative-oncogene-fusion-freq.jsonl.gz`.

Table two - `putative-oncogene-fused-gene-freq.jsonl.gz`
The overall calculation logic is identical to table one. 
The difference is instead of calculating the frequency for each unique fusion, we are calculating the frequency for each unique gene that is present within a fusion (from 1A, 1B, 2A or 2B in a particular fusion) in different tumor types.
```